test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -394,6 +394,20 @@
         (doseq [f files] (is (= "testable string" (slurp f))
                              "File was replaced"))))))
 
+(deftest initialize!-test-with-keylength-in-settings
+  (let [settings (assoc (testutils/ca-settings (ks/temp-dir)) :keylength 768)]
```

Comment:
It's not possible to use `ca-sandbox!` here because we are changing the keylength, so we need to change the public/privage key pair. This is done only if those files don't exist at first, hence we need to recreate the CA from scratch.